### DISCUSSION TITLE
xload扩展问题讨论

### DISCUSSION THREAD

**Discussion by wujinpai** [1Panel-dev/1Panel#4360] [Q&A]
#4311 问题讨论延续，![image](https://github.com/1Panel-dev/1Panel/assets/43784195/fd1796b0-bb00-4b4f-9cfb-e33eb087f8e5)

我找到了xload扩展官方支持人员，并把自己的1p面板给了他测试，经过他测试回复的结果

**Comment by wujinpai** [comment]
是 #4326 噢😅

**Comment by wanghe-fit2cloud** [comment]
https://github.com/1Panel-dev/1Panel/discussions/4372
